### PR TITLE
Use Codecov instead of coveralls

### PR DESCRIPTION
**Pull request by huonw**

### CODE REVIEW

**Comment by autohuonw** [comment]
@huonw: no appropriate reviewer found, use r? to override

**Comment by huonw** [comment]
bors try

**Comment by huonw** [comment]
bors try

File: primal-sieve/src/streaming/mod.rs [hunk 1]
```diff
@@ -352,4 +352,27 @@ mod tests {
             }
         }
     }
+
+    // These are designed to specifically test the medium sized and
+    // large prime sieving.
```
Comment (cuviper) [inline primal-sieve/src/streaming/mod.rs]: These are dealing with `usize`, so your larger values should either be scaled back a little, or gate on `target_pointer_width = "64"`.

Comment (cuviper) [reply primal-sieve/src/streaming/mod.rs]: IMO `deny(warnings)` is too annoying, as new rustc may add new warnings, but it's your call.

Comment (huonw) [reply primal-sieve/src/streaming/mod.rs]: This is a relatively small code base and deny(warnings) won't affect downstream users, so I'm happy to try it for a while, and revisit if it does get too annoying.

**Comment by huonw** [comment]
bors r=cuviper

**Pull request merged**